src/js/components/FragmentList/fragmentList.jsx

```diff
@@ -56,46 +62,106 @@ export function sortFragmentsByStatus(fragments) {
   return sortedFragments;
 }
 
+const sortingOptions = {
+  status: 'status',
+  id: 'id',
+  type: 'type',
+  time: 'time',
+};
+
 const FragmentList = ({ tabId }) => {
   const data = useSelector((state) => state.pageData[tabId].fragments);
   const parsedData = mapDataToComponents(data, tabId);
   const [fragments, setFragments] = useState(parsedData);
+  const [currentSorting, setCurrentSorting] = useState(null);
 
   const typeSortComparator = (a, b) => a.props.type.localeCompare(b.props.type);
   const idSortComparator = (a, b) => a.props.id.localeCompare(b.props.id);
+  const timeSortComparator = (a, b) => a.props.time - b.props.time;
 
   return (
     <FragmentListWrapper>
+      <h1>List of fragments</h1>
+
       <SortingWrapper>
-        <SortingButton
-          status
-          onClick={() => setFragments(sortFragmentsByStatus(fragments))}
+        <StatusSortingButton
+          onClick={() => {
+            if (!(currentSorting === sortingOptions.status)) {
```

Comment:
Wouldn't be `(currentSorting !== sortingOptions.status)` sufficient? Or How about reversing the logic, first do what should be done when `(currentSorting === sortingOptions.status)`?